measures/views.py

```diff
@@ -1098,6 +1099,51 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class MeasuresCreateProcessQueue(TemplateView, ListView):
+    """UI endpoint for bulk creating Measures process queue."""
+
+    # permissions?
+    template_name = "measures/create-process-queue.jinja"
+    filter_by_fields = ["all", "processing", "completed", "failed"]
+    sort_by_fields = [
+        "workbasket_id",
+        "submitted_date",
+        "submitted_by",
+        "status",
+        "object_count",
+        "action",
+    ]
+
+    def get_queryset(self):
+        queryset = MeasuresBulkCreator.objects.all()
+        return queryset
```

Comment:
We probably want something like the following so that we only get those creation tasks that are associated EDITING workbaskets:
```suggestion
        return MeasuresBulkCreator.objects.filter(workbasket__status=WorkflowStatus.EDITING)
```

Reply:
Resolved :)